Automattic/crowdsignal-ui
# Add placeholder to Text Question and Text Input blocks

## Summary

The purpose of this PR is to include a `placeholder` option on the Sidebar for the Text Question and Text Input block.

Ref: c/hKFZL7jl-tr

## Test Instructions
* Open or create a new project
* Add a Text Question and a Text Input block
* On the Sidebar of each block, you should see a new Placeholder option
* By setting this value, you should see it on both, the Editor and the Project Renderer
